docs/pages/config.mdx

```diff
@@ -5,19 +5,66 @@ This file needs to be named `mud.config.ts` and be in the same folder as your `f
 
 The config is used to define:
 
-- The tables in your project in the `tables` object of your configuration.
-- The [namespace](/world/namespaces-access-control) that [systems](/world/systems) and tables will be deployed in.
+- The tables in your project.
+- The [namespace(s)](/world/namespaces-access-control) that [`System`s](/world/systems) and tables will be deployed in.
```

Comment:
```suggestion
- The [namespace(s)](/world/namespaces-access-control) that [systems](/world/systems) and tables will be deployed in.
```
let's use a regular noun for systems throughout rather than a proper noun or code snippet